### DISCUSSION TITLE
Win key is disabled when playing multi and game is not in focus

### DISCUSSION THREAD

**Discussion by Susko3** [ppy/osu#16147] [Q&A] [answered]
Since multiplayer can't be paused, it's always in gameplay, even when the game is not in focus / when you're alt-tabbed away.

Doesn't make sense to block input while the game is not focused. Should be a quick fix to [`GameplayWinKeyBlocker`](https://github.com/ppy/osu/blob/master/osu.Desktop/Windows/GameplayWinKeyBlocker.cs).

`ILocalUserPlayInfo.IsPlaying` is also used in [`HighPerformanceSession`](https://github.com/ppy/osu/blob/master/osu.Game/Performance/HighPerformanceSession.cs), could also update usage there, but it may be less relevant.

**Comment by bdach** [comment, marked as answer]
Would agree with fixing the win key blocker but I don't know about the high performance session stuff. It doesn't do anything currently but feels like being tabbed out shouldn't have bearing on that.

**Reply by peppy** [reply]
https://github.com/ppy/osu/pull/16168
